public/style.css

```diff
@@ -0,0 +1,27 @@
+.login{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.loginForm{
+  display: flex;
+  flex-direction: column;
+}
+
+.register{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.registerForm{
+  display: flex;
+  flex-direction: column;
+}
+
+::placeholder{
```

Comment:
You need to add input here so it's clear which placeholder you want to change.

Reply:
:+1: